x/oracle/keeper/recent_msg.go

```diff
@@ -0,0 +1,77 @@
+package keeper
```

Comment:
When is the data used? Can you explain briefly?

Reply:
This is used for refill the cache which stores price messages for aggregation:
since we have a window of more than one block to collect the prices from all validators, if this process is interrupted before the final aggregated price written to block, we would like to have some way to retrieve these messages. After we got the final price, the messages could be able to remove, so we just need the recent messages within the window.